src/pages/updates/v10-migration/design.mdx

```diff
@@ -1,145 +1,147 @@
 
-#### Layout, grid, and spacing
+### Layout, grid, and spacing
 
-The grid has been completely redesigned for v10, to align with the IBM Design Language. The official IBM grid is now 16 columns. However, to make the migration process for v9 products smoother, Carbon v10 will default to the 12-column grid, with an option to switch to the 16-column grid. Designers starting on new layouts using v10 should use the 16-column grid.
+Carbon v10 uses the 16 column [2x Grid](/guidelines/2x-grid/basics). Carbon v10 will default to the 12-column grid with an option to switch to the 16-column grid. Designers starting on new layouts using v10 should use the 16-column grid.
 
-Layout and spacing token names have also been updated in v10 but the values remain the same.
+Layout and spacing token names were updated in v10 but the values remain the same.
 
-[Layout guidelines](/guidelines/2x-grid/basics)
+<Row>
+<Column colLg={4} colMd={4} noGutterSm>
+<ResourceCard
+    subTitle="Layout migration"
+    href="https://github.com/carbon-design-system/carbon/blob/master/docs/migration/10.x-layout.md#migrating"
+    >
 
-[v10 grid demo](https://carbon-elements.netlify.com/grid/examples/preview/)
+<MdxIcon name="github" />
 
-[v10 spacing migration](https://github.com/carbon-design-system/carbon/blob/master/docs/migration/10.x-layout.md#migrating)
+  </ResourceCard>
+</Column>
+</Row>
 
 ## Design migration strategy
 
-#### Update your kit
-
-The Carbon design kit now lives in a Sketch library, which can be updated on a regular basis by the Carbon team. If you're working with a local document, visit the [Getting started](/get-started/design) guide and get set up with the new libraries.
+The fastest approach to v10 migration starts with the front-end developer. Developers should update the product's front end code to v10 and conduct a visual review with team designers. The transition should not cause any major breaks in the UI but there will likely be layout and color issues. Product teams using v9 components in their code should see a mostly seamless code transition.
 
-The components and elements in the kit have been completely revised from v9. Be sure to note the changes to the color palette, spacing and layout, and iconography in the kit.
-
-#### "Code-first" approach
-
-The fastest approach to v10 migration starts with the front-end developer. The developer, with help from the migration guidelines found on this site, will update the product's front end code to v10. This should not cause any major breaks in the UI, but there might be some funky looking stuff that needs to be resolved by the design team. Product teams using v9 components in their code should see a mostly seamless transition.
-
-The product design team should review the updated UI with a critical eye and a solid knowledge of the v10 design guidelines. Throughout the process, ask yourself:
+Throughout visual review and iteration process, ask:
 
 - Do the fonts look right?
```

Comment:
I know these questions are old but they feel a little strange to me. How would someone know if the fonts and colors didn't look correct? I would maybe say "Are you using the type tokens?" or  "Are you using the color tokens correctly?" Because those do have specific use-cases that can be measured against and enforce the use of tokens. Not a stop-ship comment though.